mockrunner-servlet/pom.xml

```diff
@@ -22,20 +22,14 @@
 		<dependency>
 			<groupId>com.mockrunner</groupId>
 			<artifactId>mockrunner-core</artifactId>
-			<version>2.0.4-SNAPSHOT</version>
-			<scope>provided</scope>
 		</dependency>
 		<dependency>
 			<groupId>com.mockrunner</groupId>
 			<artifactId>mockrunner-core</artifactId>
-			<version>2.0.4-SNAPSHOT</version>
-			<type>xml</type>
-			<classifier>features</classifier>
```

Comment:
Maybe leave these two lines, and in the parent pom define the version.

Reply:
See above